csss-site/src/events/frosh/urls.py

```diff
@@ -45,5 +45,9 @@
     url(r'^2022$', frosh2022.index, name="Frosh Week 2022 Entrance"),
     url(r'^2022/frosh$', frosh2022.frosh, name="Frosh Week 2022"),
     url(r'^2022/secret$', frosh2022.secret, name="Not A Secret"),
+    # url(r'^2023$', frosh2023.index, name="Frosh Week 2023 Entrance"),
+    # url(r'^2023/frosh$', frosh2023.frosh, name="Frosh Week 2023"),
+    url(r'^2023/secret$', frosh2023.secret, name="Not A Secret"),
+    url(r'^2023/sponsor$', frosh2023.sponsor, name="For Sponsors"),
```

Comment:
shouldn't this end with `sponsors`? it's not like we'll have only 1 of them right? and the function should also be `sponsors`

Reply:
in addition to this, in alot of places in your code, you use `sponsor` [like in folder names] when it should really be `sponsors` imo

Reply:
this is the "sponsor site". I see what you mean by `sponsors` making sense, but the distinction is negligible

Reply:
this is the site sponsors go to to see what the event is or the site that showcases our sponsors?

Reply:
this is the site that faces external companies an encourages them to sponsor us.

Reply:
ah, gotcha